backend/pkg/scheduler.go

```diff
@@ -1,14 +1,54 @@
 package main
 
-import "github.com/grafana/grafana-plugin-sdk-go/backend/log"
+import (
+	"fmt"
+
+	"github.com/360EntSecGroup-Skylar/excelize"
+	"github.com/grafana/grafana-plugin-sdk-go/backend/log"
+	"gopkg.in/gomail.v2"
+)
 
 type EmailConfig struct {
 	email    string
 	password string
 }
 
+type SmtpServer struct {
+	host string
+	port string
+}
+
+func (smtpServer *SmtpServer) Address() string {
```

Comment:
Another very very small nit. May be wrong, but I _think_ user methods are usually `camelCase` in go (I think to distinguish from system/library methods which are often `CapitalCase`).

Reply:
Ah yup, ive since found that exposed methods are capital